plutus/budget_manager/gcp_helper.py

```diff
@@ -0,0 +1,283 @@
+from constants import APP
+import copy
+import logging
+import markus
+from protobuf_to_dict import protobuf_to_dict
+import sys
+
+metrics = markus.get_metrics(APP + ".gcphelper")
+
+class GcpHelper:
+    '''
+    This class uses GCP billing budget and resource manager clients to get, update, and create budgets
+    '''
+
+    def __init__(self, billing_client, rm_client):
+        self.billing_client = billing_client
+        self.rm_client = rm_client
```

Comment:
I've been reading this is `remove_client` instead of `resource_manager_client`.

Reply:
done